docs/adminguide/hosting/index.rst

```diff
@@ -0,0 +1,265 @@
+.. _docker compose: https://docs.docker.com/compose/
+
+.. include:: parsec-server.docker.yaml
+  :code: yaml
+
+It will setup 4 services:
+
++---------------------+----------------------------+
+| Service name        | Description                |
++=====================+============================+
+| ``parsec-postgres`` | The database PostgreSQL    |
++---------------------+----------------------------+
+| ``parsec-s3``       | The Object Storage service |
++---------------------+----------------------------+
+| ``parsec-smtp``     | Provide a mock SMTP server |
++---------------------+----------------------------+
+| ``parsec-backend``  | The parsec backend server  |
++---------------------+----------------------------+
+
+Starting the infra
+==================
+
+The docker containers can be started as follow:
+
+.. code-block:: bash
+
+  docker compose -f parsec-server.docker.yaml up
+
+Initial infra setup
+===================
+
+On the first start of the infra, we need to do a one-time configuration on the database and s3.
+
+Applying the database migration
+-------------------------------
+
+(optional) You can verify that the database is accessible with:
+
+.. code-block:: bash
+
+  set -a
+  source parsec-db.env
+  docker exec -t parsec-postgres psql 'postgresql://DB_USER:DB_PASS@0.0.0.0:5432/parsec' -c "\conninfo"
+  You are connected to database "parsec" as user "parsec" on host "0.0.0.0" at port "5432".
+
+To bootstrap the database we just need to apply the migrations with:
+
+.. code-block:: bash
+
+  docker compose -f parsec-server.docker.yaml run parsec-backend migrate
+
+Create the S3 Bucket
+--------------------
+
+Access the console at https://127.0.0.1:9090, you will need to use the credential specified in the ``docker-compose`` file at :yaml:`services.parsec-s3.environment.MINIO_ROOT_{USER,PASSWORD}`.
+
+Go to https://127.0.0.1:9090/buckets/add-bucket to create a new bucket named ``parsec`` with the features ``object locking`` toggled on.
+
+After that you will need to restart the ``parsec-backend`` (that likely exited because it wasn't able to access the S3 bucket):
+
+.. code-block:: bash
+
+  docker compose -f parsec-server.docker.yaml restart parsec-backend
+
+Test the SMTP configuration & server
+------------------------------------
+
+You can test ``mailhog`` with:
+
+.. include:: ping-mailhog.sh
+  :code: bash
+
+You can then check the email is present in the web interface at http://127.0.0.1:8025
+
+First step with the parsec infra
+********************************
+
+Create the first organization
+=============================
+
+.. code-block:: bash
+
+  set -a
+  source parsec-admin-token.env
+  export SSL_CAFILE=$PWD/custom-ca.crt
+  parsec.cli core create_organization --addr parsec://127.0.0.1:6777 <orgname>
+
+.. note::
+
+   Change ``<orgname>`` to the organization's name that suit you.
+
+Save the link after ``Bootstrap organization url:`` you will need it to create the first user (owner) of the organization.
```

Comment:
```suggestion
Save the link after ``Bootstrap organization url:`` you will need it to create the first user (administrator) of the organization.
```

Reply:
The first user of the owner of the organization :thinking:

Reply:
What I meant in my suggestion is that when you bootstrap an organization, the first user will be the _administrator_, not the _owner_ (which is a concept related to the workspaces).

In fact, we can even avoid mentioning the user creation, just the bootstrap process:

```suggestion
Save the link after ``Bootstrap organization url:`` you will need it to start the bootstrap process for the new organization.
```